src/@commitUI/components/FileUpload/index.tsx

```diff
@@ -99,7 +101,8 @@ export const FileUpload = ({
   const hasFile = acceptedFiles.length || file
 
   return (
-    <div className={styles.container}>
+    <div className={styles.container}
+      style={disabled ? {pointerEvents: "none", opacity: "0.5"}:{}}>
```

Comment:
For these, can you change it to use the `classnames` library instead? This can be done roughly like this:
``` 
import cx from "classnames"

...

const containerCn = cx(
    styles.container,
    {
      [styles.disabled]: disabled
    }
)

...

return (
  <div className={containerCn}>
```
So you will have to move the `{pointerEvents: "none", opacity: "0.5"}` to the module.scss file with the classname disabled, as we are using `styles.disabled` on the cx().

The reason being inlines styles are not really good for performance, as on each re-rendering the style object will be recomputed.

Reply:
This is a weird modification to make, in the sense that I need to modify the raw css file to get it working, but not the other one, strangely enough. Not sure if it requires an update to the commitUI package or smth, idk.